### DISCUSSION TITLE
DR25 roasting machine data collection

### DISCUSSION THREAD

**Discussion by KimSungHo369** [artisan-roaster-scope/artisan#1584] [Q&A]
1. I would like to ask how long it can last and save CSV files in real time when I can leave csv files on my local PC with Artisan

2. There are only five types of data: Bean Temperure, Drum Inlet Temperature, Drum Outlet Temperature, Fuel% and Air% and I would like to ask if we can check the operation status of the facility or the operation time

**Comment by MAKOMO** [comment]
Do you refer to a Dietrich roasting machine by the DR25 in the title?

I might not understand your questions correctly. It would be helpful if you could explain a bit the background of your questions and your the targets you are trying to achieve. I will try to answer anyhow below.

> 1. I would like to ask how long it can last and save CSV files in real time when I can leave csv files on my local PC with Artisan

The native profile file format of Artisan is not CSV, however, Artisan can export recorded profiles in CSV format. Data is never saved by Artisan in real time, but data is collected in real time to the app memory which can be saved as a profile at the end of a batch into a native .alog file or exported to a CSV file. The size of the recording and profile files is only limited by the memory and disk space of your computer

> 2. There are only five types of data: Bean Temperure, Drum Inlet Temperature, Drum Outlet Temperature, Fuel% and Air% and I would like to ask if we can check the operation status of the facility or the operation time

Which facility? If you are referring to a Diedrich DR25 roasting machine, Artisan is just reading the data provided by the machine, about what you list (which varies from machine to machine based on the supported data and installed equipment). Artisan does not has more information than that on th eoperation status. As Artisan is only recording data for roasting batches and not continuously it has only limited information on the machines operation time. Artisan also does not compute the total recording time.

Please excuse me if I did not understand your questions correctly and thus my answers do not serve you well.
